### PR TITLE
Make `TimingModel.params` and `TimingModel.ordered_params` identical.

### PR DESCRIPTION
**Pull request by abhisrkckl**
See #1295

These parameter lists had different orders because Python dicts were unordered until Python 3.7. Since this is no longer the case, `TimingModel.params` and `TimingModel.ordered_params` have no reason to be different anymore.

### CODE REVIEW

**Comment by dlakaplan** [comment]
Would it be better to just remove the `ordered_params` and make all instances `params`?

**Comment by abhisrkckl** [comment]
I think we should keep it for the time being since other people's codes may depend on it. We can deprecate it in the future and then remove it once people get enough time to change their code.

File: src/pint/models/timing_model.py [hunk 1]
```diff
@@ -500,20 +499,30 @@ def __getattr__(self, name):
...
+    def params_ordered(self):
+        """List of all parameter names in this model and all its components.
+        This is the same as `params`."""
+
+        # Historically, this was different from `params` because Python
+        # dictionaries were unordered until Python 3.7. Now there is no reason for
+        # them to be different.
+
+        warn(
+            "`TimingModel.params_ordered` is now deprecated and maybe removed in the future. "
```
Comment (dlakaplan) [inline src/pint/models/timing_model.py:511]: "maybe" -> "may be"

Comment (dlakaplan) [reply src/pint/models/timing_model.py:511]: also, shouldn't there be some sort of way that the two lines of the warning are connected?  Is this actually tested?

Comment (dlakaplan) [reply src/pint/models/timing_model.py:511]: Is this best done as a generic `warn` or https://docs.python.org/3/library/exceptions.html#DeprecationWarning?

Comment (abhisrkckl) [reply src/pint/models/timing_model.py:511]: > also, shouldn't there be some sort of way that the two lines of the warning are connected? Is this actually tested?

Python treats such strings as a single string with no newline in between. In a script,
```
"foo"
"bar"
```
is the same as `"foobar"`.

Comment (abhisrkckl) [reply src/pint/models/timing_model.py:511]: > Is this best done as a generic warn or

The `warn` function does invoke a `DeprecationWarning`.

Comment (dlakaplan) [reply src/pint/models/timing_model.py:511]: ok, I didn't read down far enough.

**Comment by dlakaplan** [comment]
OK, I think I can merge this when you are done.

**Comment by abhisrkckl** [comment]
I am done.

**Pull request merged**